src/service/xhr-impl.js

```diff
@@ -158,6 +163,15 @@ export class Xhr {
         user().assert(false, `Response must contain the` +
             ` ${ALLOW_SOURCE_ORIGIN_HEADER} header`);
       }
+
+      // If the `AMP-Redirect-To` header is set. Redirect the user to the
+      // given URL.
+      const redirectTo = response.headers.get(REDIRECT_TO_HEADER);
+      if (redirectTo) {
+        assertAbsoluteHttpOrHttpsUrl(redirectTo);
```

Comment:
like "The AMP-Redirect-To header value must be an absolute URL starting with https://. Found %s"